### PR TITLE
Create print-error-chrome.md

### PR DESCRIPTION
**Pull request by nelci592**

### CODE REVIEW

File: knowledge-base/print-error-chrome.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+		<td>Product</td>
+		<td>Progress® Telerik® Reporting</td>
+	</tr>
+</table>
+
+
+## Description
+After the Google Chrome update to version 77.0.3865.75, the print dialog doesn't open. In this article, you can find a workaround which makes possible downloading the report as a PDF file.
+
+## Solution
```
Comment (silviyaboteva) [inline knowledge-base/print-error-chrome.md]: Workaround instead

File: knowledge-base/print-error-chrome.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+```JavaScript
+$("#reportViewer1")
+	.telerik_ReportViewer({                  
+       serviceUrl: "/api/reports",
+		...
+       printMode: telerikReportViewer.PrintModes.FORCE_PDF_FILE
+		},
+...
+```
```
Comment (silviyaboteva) [inline knowledge-base/print-error-chrome.md]: Do we have to mention the public item here as well? Or not

File: knowledge-base/print-error-chrome.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+---
+title: Print dialog doesn't open in Google Chrome 77.0.3865.75 
+description: Since the Google Chrome update to version 77.0.3865.75 the print functionality in the web-based viewers doesn't work
+type: troubleshooting
+page_title: Print functionality does not work in Google Chrome 77.0.3865.75 
+slug: print-error-chrome
+position: 
+tags: HTML5Viewer, MVCReportViewer, RESTService, WebFormsViewer, AngularViewer
```
Comment (todorarabadzhiev) [inline knowledge-base/print-error-chrome.md]: remove REST Service

File: knowledge-base/print-error-chrome.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+	</tr>
+</table>
+
+
+## Description
+After the Google Chrome update to version 77.0.3865.75, the print dialog doesn't open. In this article, you can find a workaround which makes possible downloading the report as a PDF file.
+
+## Solution
+
+By setting **FORCE_PDF_FILE** [printMode](../reporting/html5-report-viewer-api-printmodes) option of the report viewer, the widget will always export the report document to a PDF file with the special print script. For more information, check [Printing Reports](../reporting/html5-report-viewer-direct-print) article.
```
Comment (todorarabadzhiev) [inline knowledge-base/print-error-chrome.md]: remove 'reporting' from url

File: knowledge-base/print-error-chrome.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+</table>
+
+
+## Description
+After the Google Chrome update to version 77.0.3865.75, the print dialog doesn't open. In this article, you can find a workaround which makes possible downloading the report as a PDF file.
+
+## Solution
+
+By setting **FORCE_PDF_FILE** [printMode](../reporting/html5-report-viewer-api-printmodes) option of the report viewer, the widget will always export the report document to a PDF file with the special print script. For more information, check [Printing Reports](../reporting/html5-report-viewer-direct-print) article. 
+For example in the [HTML5 Report viewer](../reporting/html5-report-viewer), you need to add the following line in the initialization of the viewer:
```
Comment (todorarabadzhiev) [inline knowledge-base/print-error-chrome.md]: remove 'reporting' from url

**Pull request merged**